data/_models/shared/Organisations.csv

```diff
@@ -1,12 +1,12 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Organisations,,,,,,,,,,,"org:Organization,foaf:Agent",Research departments and research groups. Null values will be generated from RoR,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
-Organisations,,resource,ref,1,TRUE,,Resources,,,,dcterms:isPartOf,Resource that this organisation is part of,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
+Organisations,,,,,,,,,,,foaf:Agent,Research departments and research groups. Null values will be generated from RoR,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
+Organisations,,resource,ref,1,TRUE,,Resources,,,,,Resource that this organisation is part of,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
 Organisations,,id,,1,TRUE,,,,,,dcterms:identifier,Internal identifier,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
 Organisations,,pid,hyperlink,,,,,,,,dcterms:identifier,"Persistent identifier, ideally from ROR","DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
-Organisations,,name,text,,TRUE,,,,,,"dcterms:title,foaf:name",Name used in (inter)national projects,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
-Organisations,,acronym,,,,,,,,,dcterms:identifier,"Acronym, if applicable","DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts",
```

Comment:
dit ook, waarom dit allemaal weghalen, heb je er last van?

Reply:
dct:identifier is not used within foaf:Agent or org:Organization class and acronym is not an identifier, the pid/ror is. changed to skos:altLabel.

dcterms:title is not used within foaf:Agent or org:Organization class. changed to skos:prefLabel